validator/keymanager/v2/remote/doc.go

```diff
@@ -0,0 +1,59 @@
+/*
+Package remote defines an implementation of an on-disk, EIP-2335 keystore.json
+approach towards defining validator accounts in Prysm. A validating private key is
+encrypted using a passphrase and its resulting encrypted file is stored as a
+keystore.json file under a unique, human-readable, account namespace. This direct keymanager approach
+relies on storing account information on-disk, making it trivial to import, export and
+list all associated accounts for a user.
+
+Package remote defines a keymanager implementation which connects to a remote signer
+server via gRPC. The connection is established via TLS using supplied paths to
+certificates and key files and allows for submitting remote signing requests for
+eth2 data structures as well as retrieving the available signing public keys from
+the remote server.
+
+Remote sign requests are defined by the following protobuf schema:
```

Comment:
Does this need to be in the comment? Could be annoying to maintain.

Reply:
It should be fine given how simple the proto message is. If it becomes more complex, agreed we should instead link to it. However, I do think it gives a nice picture to someone looking to write a a remote signer server as to what the protos should look like.